homeassistant/components/atome/sensor.py

```diff
@@ -111,11 +111,14 @@ def is_connected(self):
         """Return latest active power value."""
         return self._is_connected
 
-    @Throttle(LIVE_SCAN_INTERVAL)
-    def update_live_usage(self):
-        """Return current power value."""
-        try:
-            values = self.atome_client.get_live()
+    def _retrieve_live(self):
+        error_during_retrieve = False
```

Comment:
I think you can drop this if you inline below...
```suggestion
```

Reply:
No this one, the code will be become tricky

Reply:
Sorry, I 've not seen below (ok then)  ;-)